test/jdk/java/awt/dnd/RejectDragDropActionTest.java

```diff
@@ -74,27 +72,30 @@ public void drop(DropTargetDropEvent dtde) {
         };
     private final DropTarget dt = new DropTarget(frame, dtl);
 
-    public static void main(String[] args) throws InterruptedException,
-            InvocationTargetException, AWTException {
+    public static void main(String[] args) throws Exception {
         EventQueue.invokeAndWait(() -> {
             frame = new Frame("RejectDragDropActionTest");
             ds = DragSource.getDefaultDragSource();
             dgl = dge -> dge.startDrag(null, new StringSelection("OOKK"));
-            dgr = ds.createDefaultDragGestureRecognizer(frame, DnDConstants.ACTION_COPY, dgl);
-            frame.setBounds(100, 100, 200, 200);
+            dgr = ds.createDefaultDragGestureRecognizer(frame,
+                    DnDConstants.ACTION_COPY, dgl);
+            frame.setSize(200, 200);
+            frame.setLocationRelativeTo(null);
             frame.setVisible(true);
         });
 
         try {
             Robot robot = new Robot();
             robot.waitForIdle();
-            robot.delay(FRAME_ACTIVATION_TIMEOUT);
+            robot.delay(DELAY_TIME);
```

Comment:
Does adding `robot.autoWaitForIdle(true)` additionally increase the stability of the test since the test has subsequent DnD events performed by the robot?

Reply:
Sure, I just added it. Previously when I initially worked on this test, setAutoWaitForIdle had a weird interaction with how the test was setup. Since the test used a delay of 50ms between each of the movements when dragging the selection, it caused long delays. Looks like this is resolved now, and the method doesn't add any unreasonable delays anymore.

Reply:
Sounds good!